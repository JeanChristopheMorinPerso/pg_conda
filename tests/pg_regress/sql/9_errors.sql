BEGIN;
CREATE EXTENSION pg_conda;

SELECT 'asd }{}'::condaversion;

ROLLBACK;
