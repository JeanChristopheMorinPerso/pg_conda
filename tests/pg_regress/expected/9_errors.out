BEGIN;
CREATE EXTENSION pg_conda;
SELECT 'asd }{}'::condaversion;
ERROR:  failed to parse version "asd }{}". It is probably not a valid conda version. Error: malformed version string 'asd }{}': encountered more characters but expected none at character 8
ROLLBACK;
